bookwyrm/templates/shelf/shelf.html

```diff
@@ -209,7 +210,12 @@ <h2 class="title is-3">
             </tbody>
         </table>
         {% else %}
-        <p><em>{% trans "This shelf is empty." %}</em></p>
+        <p><em>{% if shelves_filter_query %}
+                    {% trans "We couldn't find any books that matched" %} "{{ shelves_filter_query }}"
```

Comment:
This will need to be a blocktrans element with the query included, so that languages with different word order can translate it correctly. It should look something like:
```
{% blocktrans trimmed %}
We couldn't find any books that matched {{ shelves_filter_query }}
{% endblocktrans %}
```

Reply:
Done!